src/components/AttachmentModal.tsx

```diff
@@ -279,19 +308,20 @@ function AttachmentModal(props) {
      * @param {Object} _data
      */
     const validateAndDisplayFileToUpload = useCallback(
-        (_data) => {
+        // eslint-disable-next-line @typescript-eslint/naming-convention
+        (_data: Data) => {
             if (!isDirectoryCheck(_data)) {
                 return;
             }
-            let fileObject = _data;
-            if (typeof _data.getAsFile === 'function') {
+            let fileObject: Data | null = _data;
```

Comment:
Why do we need `| null` in this line?

Reply:
I think this is something which I added earlier when `Data` had some slightly different type and then this null was necessary but its not needed anymore